docs/lib_wavefile-creator.js.html

```diff
@@ -226,7 +226,7 @@ <h1 class="page-title">lib/wavefile-creator.js</h1>
     }
     if (
       (info.sampleRate == 44100 || info.sampleRate == 22050) &amp;&amp;
-      info.padding
+      !info.padding
```

Comment:
This is the issue that needed to be changed - was checking for if there was padding instead of NOT padding